### PR TITLE
Migrate application to GCP

### PR DESCRIPTION
**Pull request by janzh**
Migratet data in database from dev-fss and testet functionality in dev. Will migrate prod later.

### CODE REVIEW

File: .nais/naiserator-dev.yaml [hunk 1]
```diff
@@ -46,26 +45,33 @@ spec:
         - application: syfomodiaperson
           namespace: teamsykefravr
           cluster: dev-fss
+    outbound:
+      external:
+        - host: "kafka-schema-registry.nais-q.adeo.no"
+        - host: "syfo-tilgangskontroll.prod-fss-pub.nais.io"
```
Comment (JMLindseth) [inline .nais/naiserator-dev.yaml:51]: prod-fss -> dev-fss?

File: .nais/naiserator-prod.yaml [hunk 2]
```diff
@@ -46,26 +45,33 @@ spec:
         - application: syfomodiaperson
           namespace: teamsykefravr
           cluster: prod-fss
+    outbound:
+      external:
+        - host: "kafka-schema-registry.nais.adeo.no"
+        - host: "syfo-tilgangskontroll.dev-fss-pub.nais.io"
```
Comment (JMLindseth) [inline .nais/naiserator-prod.yaml:51]: dev-fss -> prod-fss?

File: .nais/naiserator-dev.yaml (hunk 1 above)
Comment (janzh) [reply .nais/naiserator-dev.yaml]: Fikset.

File: .nais/naiserator-prod.yaml (hunk 2 above)
Comment (janzh) [reply .nais/naiserator-prod.yaml]: Fikset.

**Review by JMLindseth — APPROVED** [review]

**Pull request merged**